pinot-core/src/main/java/org/apache/pinot/core/realtime/stream/StreamConfigProperties.java

```diff
@@ -89,6 +89,11 @@
    */
   public static final String SEGMENT_FLUSH_DESIRED_SIZE = "realtime.segment.flush.desired.size";
 
+  /**
+   * The initial num rows to use for segment size auto tuning. By default 100_000 is used.
+   */
+  public static final String SEGMENT_INITIAL_ROWS_THRESHOLD = "realtime.segment.initial.rows.threshold";
```

Comment:
I suggest "realtime.segment.flush.initial.rows.threshold". What do you think?

Reply:
Yes, sounds good. Changed it